### PR TITLE
Fix/balancing

### PR DESCRIPTION
**Pull request by latedude2**

### CODE REVIEW

File: Assets/Scripts/HoldObject.cs [hunk 1]
```diff
@@ -70,6 +71,7 @@ private void CarryObject()
     IEnumerator Release()
     {
         setPlayerControl(false);
+        GetComponent<PlayerMovement>().isKoalaPickedUp = false;
```
Comment (Jokubas126) [inline Assets/Scripts/HoldObject.cs]: please use `IsHolding` property in this script from within PlayerMovement. It's meant to work that way

File: Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -11,6 +11,8 @@ public class PlayerMovement : MonoBehaviour
     public float rotateSpeed = 250.0f;
     public int noMovementRotSteps = 16;
     public bool isUnderPlayerControl = true;
+    public bool isKoalaPickedUp = false;
```
Comment (Jokubas126) [inline Assets/Scripts/PlayerMovement.cs]: you won't need this parameter then

**Review by Jokubas126 — CHANGES_REQUESTED** [review]
Please correct 🙏

File: Assets/Scripts/HoldObject.cs (hunk 1 above)
Comment (latedude2) [reply Assets/Scripts/HoldObject.cs]: ah, missed it

**Review by Jokubas126 — APPROVED** [review]
nic 🥔

**Pull request merged**